calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/EventManagement.java

```diff
@@ -787,25 +784,24 @@ public void inputRecurringInfoEvent(repeatType repeatType,
                                       String... option) {
     info("Add recurring information");
     String occurence = option.length > 0 ? option[0] : "";
-    ELEMENT_REPEAT_SWITCH_LABEL.click();
     if (endRepeat != null) {
       switch (endRepeat) {
-      case After:
-        info("Check After option");
-        evt.check($(byId("endAfter")), 2);
-        if (occurence != "")
-          evt.type(ELEMENT_END_AFTER_NUMBER, option[0], true);
-        break;
-      case ByThisDate:
-        info("Check By this date option");
-        evt.check(ELEMENT_BY_THIS_DATE_END_RECURRING_EVENT, 2);
-        if (occurence != "")
-          evt.type(ELEMENT_DATE_TIME_PICKER, option[0], true);
-        break;
-      case Never:
-        info("Check never option");
-        evt.check(ELEMENT_NEVER_END_RECURRING_EVENT, 2);
-        break;
+        case After:
+          info("Check After option");
+          evt.check(byId("endAfter"), 2);
+          if (occurence != "")
+            // evt.type(ELEMENT_END_AFTER_NUMBER, option[0], true);
```

Comment:
Should be removed if it is not needed anymore (as well as the if).

Reply:
Done.